server/utils/translatingFunction.js

```diff
@@ -0,0 +1,73 @@
+import asyncHandler from "../middleware/asyncHandler.js";
+
+import Song from "../models/Song.js";
+
+import { translateLyricsByOpenAi } from "./openAiTranslation.js";
+
+async function searchSongByLyrics(originalLang, lyrics) {
+    try {
+      const query = {
+        originalLang: originalLang,
+        $or: [
+          { 'lyrics.hebrew': lyrics },
```

Comment:
change this check. the lyrics don't have to be equal to "lyrics" param. if there is something inside - return it

Reply:
But, I still don't understand clear how I should find a song if it user will send to function original lyrics to get translation. And  my idea is to find song by original lyrics. so that lyrics will be always the same because we will get them, from lyricsAPI and not by openAI translation.

Reply:
where is the change?